Add camera view (#428)
MicrosoftDocs/CommunityToolkit: docs/maui/views/camera-view.md

---
title: CameraView - .NET MAUI Community Toolkit
author: bijington

> description: "The CameraView provides the ability to connect to a camera, display a preview from the camera and take photos."

Author: Unless I've missed something I believe we only support image capture in v1

> description: "The CameraView provides the ability to connect to a camera, display a preview from the camera and take photos."

Reviewer: In that case we should remove "recording videos" on line 10 😄

> description: "The CameraView provides the ability to connect to a camera, display a preview from the camera and take photos."

Author: How does it look now?

> description: "The CameraView provides the ability to connect to a camera, display a preview from the camera and take photos."

Reviewer: All good! Thanks!

ms.date: 05/23/2024
---

# CameraView

> The `CameraView` provides the ability to connect to a camera, display a preview from the camera and take photos. The `CameraView` also offers the options you would expect to support taking photos and recording videos such as turning the flash on or off, saving the captured media to a file, and offering different hooks for events.

Author: ```suggestion
The `CameraView` provides the ability to connect to a camera, display a preview from the camera and take photos. The `CameraView` also offers features to support taking photos, controlling the flash, saving captured media to a file, and offering different hooks for events.
```
